docs/current/guides/541047-alternative-runtimes.md

```diff
@@ -13,15 +13,39 @@ date: "2023-04-28"
 
 This guide explains how to use Dagger with various OCI-compatible Docker alternatives.
 
-## Podman
+## Approaches
 
-### Requirements
+It is possible to run the Dagger Engine container with any other OCI-compatible container runtime, not just those which are CLI-compatible with Docker. There are two possible approaches.
+
+### Use a `docker` symbolic link
+
+By default, Dagger tries to invoke the `docker` executable. To use a different container runtime instead, create a symbolic link to it in your system path and name it `docker`. This approach is suitable for runtimes which are CLI-compatible with Docker.
+
+### Run the Dagger Engine container manually
+
+An alternative approach is to run the Dagger Engine container manually and set the `_EXPERIMENTAL_DAGGER_RUNNER_HOST` environment variable to point to the running container. If this variable is set, then Dagger will instead connect to the endpoint specified there. This approach is suitable for runtimes which are not CLI-compatible with Docker, or when using a customized Dagger Engine container.
```

Comment:
This is an experimental variable, is it OK to document it publicly?

Reply:
We already have guide that uses experimental feature, for example: https://docs.dagger.io/757394/use-service-containers/

![Screenshot 2023-08-16 at 13 59 25](https://github.com/dagger/dagger/assets/61683879/a59a4943-ded1-4042-975a-9e251778fcac)